grammar/grammar.jjt

```diff
@@ -126,18 +128,22 @@ TOKEN :
   < #PERIOD: "." >
 }
 
-/** Main production. */
-ASTStart Start() #Start : {}
+ASTStart StartCommandLine() #Start : {}
 {
-  (<WHERE>)? DisjunctionExpression() <EOF>
+  (<WHERE>)? DisjunctionExpression() "\n"
```

Comment:
@javierlores why'd you replace `<EOF>` with `"\n"` here? Shouldn't we use something that is a cross platform line separator?